pulp_rpm/app/tasks/copy.py

```diff
@@ -60,8 +78,47 @@ def find_children_of_content(content, repository_version):
             except MultipleObjectsReturned:
                 raise
 
-    # TODO: Find rpms referenced by PackageGroups,
-    # PackageGroups referenced by PackageCategories, etc.
+    # PackageCategories & PackageEnvironments resolution must go before PackageGroups
+    for packagecategory in packagecategories.iterator():
```

Comment:
Can we add a note that in the future we might be able to make this more efficient? It's not worth spending more time on it now but I'm pretty sure we could eventually get the # of queries down.

Reply:
added 'todo' comment to lower queries